subgraph/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "FleekNFA",
+  "license": "UNLICENSED",
+  "scripts": {
+    "codegen": "graph codegen",
+    "build": "graph build",
+    "deploy": "graph deploy --node https://api.studio.thegraph.com/deploy/ FleekNFA",
+    "create-local": "graph create --node http://localhost:8020/ FleekNFA",
+    "remove-local": "graph remove --node http://localhost:8020/ FleekNFA",
+    "deploy-local": "graph deploy --node http://localhost:8020/ --ipfs http://localhost:5001 FleekNFA",
```

Comment:
Deploy should be removed since it uses subgraph studio (not available for hosted service)
I'll add description to docs for the rest.

Reply:
Just to summarize our chat, create is used to create the infra needed to run a subgraph and for loca we'd need that. But for testnet it's already deployed since it's a hosted version so therefore no create needed for testnet.